integrationTests/vm/txsFee/relayedBuiltInFunctions_test.go

```diff
@@ -20,97 +20,114 @@ import (
 )
 
 func TestRelayedBuildInFunctionChangeOwnerCallShouldWork(t *testing.T) {
-	testContext, err := vm.CreatePreparedTxProcessorWithVMs(
-		config.EnableEpochs{
-			PenalizedTooMuchGasEnableEpoch: integrationTests.UnreachableEpoch,
-		})
-	require.Nil(t, err)
-	defer testContext.Close()
+	t.Run("before relayed move balance fix", testRelayedBuildInFunctionChangeOwnerCallShouldWork(integrationTests.UnreachableEpoch))
+	t.Run("after relayed move balance fix", testRelayedBuildInFunctionChangeOwnerCallShouldWork(0))
+}
 
-	scAddress, owner := utils.DoDeploy(t, testContext, "../wasm/testdata/counter/output/counter.wasm")
-	testContext.TxFeeHandler.CreateBlockStarted(getZeroGasAndFees())
-	utils.CleanAccumulatedIntermediateTransactions(t, testContext)
+func testRelayedBuildInFunctionChangeOwnerCallShouldWork(relayedFixActivationEpoch uint32) func(t *testing.T) {
+	return func(t *testing.T) {
+		testContext, err := vm.CreatePreparedTxProcessorWithVMs(
+			config.EnableEpochs{
+				PenalizedTooMuchGasEnableEpoch:   integrationTests.UnreachableEpoch,
+				FixRelayedMoveBalanceEnableEpoch: relayedFixActivationEpoch,
+			})
+		require.Nil(t, err)
+		defer testContext.Close()
 
-	relayerAddr := []byte("12345678901234567890123456789033")
-	newOwner := []byte("12345678901234567890123456789112")
-	gasLimit := uint64(1000)
+		scAddress, owner := utils.DoDeploy(t, testContext, "../wasm/testdata/counter/output/counter.wasm")
+		testContext.TxFeeHandler.CreateBlockStarted(getZeroGasAndFees())
+		utils.CleanAccumulatedIntermediateTransactions(t, testContext)
 
-	txData := []byte(core.BuiltInFunctionChangeOwnerAddress + "@" + hex.EncodeToString(newOwner))
-	innerTx := vm.CreateTransaction(1, big.NewInt(0), owner, scAddress, gasPrice, gasLimit, txData)
+		relayerAddr := []byte("12345678901234567890123456789033")
+		newOwner := []byte("12345678901234567890123456789112")
+		gasLimit := uint64(1000)
 
-	_, _ = vm.CreateAccount(testContext.Accounts, relayerAddr, 0, big.NewInt(30000))
+		txData := []byte(core.BuiltInFunctionChangeOwnerAddress + "@" + hex.EncodeToString(newOwner))
+		innerTx := vm.CreateTransaction(1, big.NewInt(0), owner, scAddress, gasPrice, gasLimit, txData)
 
-	rtxData := integrationTests.PrepareRelayedTxDataV1(innerTx)
-	rTxGasLimit := 1 + gasLimit + uint64(len(rtxData))
-	rtx := vm.CreateTransaction(0, innerTx.Value, relayerAddr, owner, gasPrice, rTxGasLimit, rtxData)
+		_, _ = vm.CreateAccount(testContext.Accounts, relayerAddr, 0, big.NewInt(30000))
 
-	retCode, err := testContext.TxProcessor.ProcessTransaction(rtx)
-	require.Equal(t, vmcommon.Ok, retCode)
-	require.Nil(t, err)
+		rtxData := integrationTests.PrepareRelayedTxDataV1(innerTx)
+		rTxGasLimit := 1 + gasLimit + uint64(len(rtxData))
```

Comment:
there are still some magic 1's here and in other places.
Could you replace with some constant/variable?

Reply:
done, replaced everywhere